src/Contact/View/page.twig

```diff
@@ -1,5 +1,23 @@
 {% include 'Common/View/header.twig' %}
 <section class="container">
-        <h1>Kontakt</h1>
+        <form method ="post" action="">   
+                <h1>NAPISZ DO NAS</h1>
+                <div>
```

Comment:
If you do not have any specific reason to keep each input in div element, please remove it. If you want to style it later, you can add classes to inputs element. Try to avoid adding additional div if they do nothing.